src/hotel/styles.ts

```diff
@@ -0,0 +1,11 @@
+export const modalStyle = {
+    position: 'absolute' as 'absolute',
```

Comment:
Why you decided to add here construction 'as'

Reply:
Purely because I got this from MUI docs just to set up some kind of modal, to work with.